sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_session.py

```diff
@@ -121,8 +121,6 @@ def get_state(self, **kwargs):
             timeout=timeout
         )
         session_state = response.get(MGMT_RESPONSE_SESSION_STATE)  # type: ignore
-        if isinstance(session_state, six.binary_type):
```

Comment:
It won't let me comment above where relevant but the type signature needs to change too.

I'd also call out that there's an async version of all this as well that'd need to change similarly.